libraries/google_bigquery_dataset.rb

```diff
@@ -79,6 +83,6 @@ def product_url
   end
 
   def resource_base_url
-    'projects/{{project}}/datasets/{{name}}'
+    'projects/{{project}}/datasets/{{dataset_id}}'
```

Comment:
Same issue in InSpec. If we move away from name (especially to a nested property), this will break.